cogs/msg.py

```diff
@@ -74,7 +76,22 @@ async def on_message(self, message):
         if len(result) == 0:
             return
 
-        await message.channel.send(result[0][1])
+        text = result[0][1]
+
+        def isUrl(url):
+            try:
+                result = urlparse(url)
+                return all([result.scheme, result.netloc])
+            except:
+                return False
+
+        rest = message.clean_content[search.end():].strip()
+        if isUrl(text) and len(rest) > 0:
```

Comment:
This should also be disableable using a config option in my opinion.

Reply:
Added a permission and per-shorthand enable/disable